package.json

```diff
@@ -40,22 +40,25 @@
     "build-electron-all": "npm run clean && npm run build-electron -- --platform=win32 && npm run build-electron -- --platform=linux && npm run build-electron -- --platform=darwin",
     "clean": "rm -rf builds config.json tasks.json coverage dist",
     "lint": "eslint *.js app test",
-    "dist": "npm run clean && electron-builder -wlm"
+    "dist": "npm run clean && electron-builder -wlm --x64 --ia32",
+    "dist-linux": "npm run clean && electron-builder -l --x64 --ia32",
+    "dist-win": "npm run clean && electron-builder -w --x64 --ia32",
+    "dist-mac": "npm run clean && electron-builder -m"
   },
   "build": {
     "appId": "angrykoala.gaucho",
     "linux": {
       "target": [
         "deb",
         "rpm",
-        "AppImage"
+        "pacman"
       ],
       "category": "Utility"
     },
     "win": {
       "target": [
         "nsis",
-        "portable"
+        "zip"
```

Comment:
Why did you remove portable?

Reply:
Packaging in zip deploys a portable version as well, it just doesn't make a first-run executable, so you launch it directly and you don't depend on any external tool in the packaging process. Although seeing it now, I think it's a good idea to switch the order between nsis and zip, so if nsis fails (happens sometimes in Linux), at least the zip target is built.

Reply:
After trying, I can see that the generated versions are:
* win-unpacked: standalone executable in the folder, this is generated always, for every system in the build
* Gaucho setup (nsis), installer for windows
* `zip`: same as unpacked but in a zip

The portable version is a single file that can be executed, while indeed, you can use the unpacked version (or the zip directly), I would like to maintain the "portable" version as a true standalone single-portable-file

Reply:
Ok, will change.